### PR TITLE
Add custom thor solo block interval

### PR DESCRIPTION
**Pull request by MakisChristou**
Tackles the following [issue](https://github.com/orgs/vechain/projects/22/views/1?filterQuery=label%3A%22App%3A+Thor%22%2C%22App%3A+VIP%22%2C%22App%3A+Thorest%22++solo&pane=issue&itemId=53228366). In this PR I basically added the option to use a command-line argument specifically for solo to allow the developer to choose a custom block interval with a minimum value of 1. I tested it and it looks like its working fine for 1 second block intervals as well as larger intervals like 100 seconds.

### CODE REVIEW

File: cmd/thor/flags.go [hunk 1]
```diff
@@ -128,6 +128,11 @@ var (
 		Name:  "on-demand",
 		Usage: "create new block when there is pending transaction",
 	}
+	blockInterval = cli.IntFlag{
+		Name:  "interval",
```
Comment (darrenvechain) [inline cmd/thor/flags.go:132]: `block-interval` maybe? I think interval is too generic

Comment (MakisChristou) [reply cmd/thor/flags.go]: What about now?

**Review by paologalligit — APPROVED** [review]
LGTM

File: cmd/thor/flags.go [hunk 2]
```diff
@@ -128,6 +128,11 @@ var (
 		Name:  "on-demand",
 		Usage: "create new block when there is pending transaction",
 	}
+	blockInterval = cli.IntFlag{
+		Name:  "block-interval",
+		Value: 10,
+		Usage: "choose a custom block interval for solo mode",
```
Comment (libotony) [inline cmd/thor/flags.go:134]: Should mention it was in `seconds`?

File: cmd/thor/flags.go [hunk 3]
```diff
@@ -128,6 +128,11 @@ var (
 		Name:  "on-demand",
 		Usage: "create new block when there is pending transaction",
 	}
+	blockInterval = cli.IntFlag{
+		Name:  "block-interval",
+		Value: 10,
```
Comment (libotony) [inline cmd/thor/flags.go:133]: The value should be checked, shouldn't be 0.

**Pull request closed without merging**